include/boost/math/distributions/logistic.hpp

```diff
@@ -145,12 +147,8 @@ namespace boost { namespace math {
           return result;
        }
        BOOST_MATH_STD_USING
```

Comment:
It looks like this is no longer necessary (likewise in `quantile`, and in the complemented versions).